skorch/net.py

```diff
@@ -141,11 +141,20 @@ class NeuralNet:
       data and should return the tuple ``dataset_train, dataset_valid``.
       The validation data may be None.
 
-    callbacks : None or list of Callback instances (default=None)
-      More callbacks, in addition to those returned by
-      ``get_default_callbacks``. Each callback should inherit from
-      :class:`.Callback`. If not ``None``, a list of callbacks is
-      expected where the callback names are inferred from the class
+    callbacks : None, str, or list of Callback instances (default=None)
+      Which callbacks to enable. There are three possible values:
```

Comment:
Since we only accept only one string option, we can be explicit here:

```suggestion
    callbacks : None, "disable", or list of Callback instances (default=None)
      Which callbacks to enable. There are three possible values:
```